source/gear/feverbreak-base.js

```diff
@@ -0,0 +1,39 @@
+const { GearTemplate } = require('../classes');
+const { dealDamage, removeModifier } = require('../util/combatantUtil');
+const { listifyEN } = require('../util/textUtil');
+const { SURPASSING_VALUE } = require('../constants');
+
+module.exports = new GearTemplate("Fever Break",
+    "Deals to a foe, the damage that is pending from any Poison and Frail on them, and then removes those debuffs",
+    "Poison and Frail not removed",
+    "Spell",
+    "Darkness",
+    200,
+    ([target], user, isCrit, adventure) => {
+        const { element } = module.exports;
+        const funnelCount = adventure.getArtifactCount("Spiral Funnel");
+        const poisons = target.getModifierStacks("Poison");
+        const frails = target.getModifierStacks("Frail");
+        let pendingDamage = (10 + 5 * funnelCount) * (poisons ** 2 + poisons) / 2 + (20 + 5 * funnelCount) * frails;
+        let cleanseText = "";
+        if (user.element === element) {
+            target.addStagger("elementMatchFoe");
+        }
+        if (!isCrit) {
+            removedDebuffs = [];
```

Comment:
```suggestion
        let removedDebuffs = [];
        if (!isCrit) {
```
As written, it probably works because it's leaving `removedDebuffs` in the global scope. Would be neater to block scope the array instead.